drivers/backup/cloud_config.json

```diff
@@ -0,0 +1,54 @@
+{
+  "cloudProviders": {
+    "aws": {
+      "default": {
+        "access_key_id": "AWS_ACCESS_KEY_ID",
+        "secret_access_key": "AWS_SECRET_ACCESS_KEY",
+        "region": "AWS_REGION"
+      }
+    },
+    "azure": {
+      "default": {
+        "subscription_id": "AZURE_SUBSCRIPTION_ID",
+        "client_id": "AZURE_CLIENT_ID",
+        "client_secret": "AZURE_CLIENT_SECRET",
+        "tenant_id": "AZURE_TENANT_ID"
+      }
```

Comment:
Please once check the method : GetAzureCredsFromEnv
AZURE_ACCOUNT_NAME and AZURE_ACCOUNT_KEY also needs to be initialized here.

Reply:
done